src/d3a/models/market/one_sided.py

```diff
@@ -146,74 +206,28 @@ def accept_offer(self, offer_or_id: Union[str, Offer], buyer: str, *, energy: in
             if time is None:
                 time = self.now
 
-            energy_portion = energy / offer.energy
             if energy == 0:
                 raise InvalidTrade("Energy can not be zero.")
-            # partial energy is requested
```

Comment:
Does it hurt to leave this comment?

Reply:
I just moved it 2 lines down to be consistent.